packages/web-components/src/components/va-radio-option/va-radio-option.tsx

```diff
@@ -52,7 +57,7 @@ export class VaRadioOption {
         onClick={() => this.handleChange()}
         role="radio"
         value={this.value}
-        id={this.label}
+        id={this.customid || this.label}
```

Comment:
Instead of giving people another prop that they have to think about, what if we just let them use `id`?

![image](https://user-images.githubusercontent.com/2008881/151897151-d56ef065-0504-4a25-b55c-ea7b90d43a44.png)

Reply:
@bkjohnson good call on this I have updated accordingly